Assets/Scripts/GameManager.cs

```diff
@@ -19,16 +20,36 @@ public class GameManager : MonoBehaviour
      //private bool transitionStarted = false;
 
      private int frameCount = 0;
-
+    
 
     Text statusText;
 
+    public static int level;
+    public GameObject ballPrefab;
+
+    private int minX;
+    private int minY;
+    private int maxX;
+    private int maxY;
+    private float minDistance;
     // Use this for initialization
     void Start()
     {
+        
         CDS = GameObject.Find ("DelayedStart").GetComponent<DelayedStartScript> ();
-        Cells = GameObject.FindGameObjectsWithTag("Cell");
+        
+        level = MainMenu.level;
+        Debug.Log(level);
```

Comment:
If you are to print it out to console, please refine the log message, otherwise removing might be an idea.